internal/services/monitor/monitor_diagnostic_setting_resource.go

```diff
@@ -462,6 +463,8 @@ func resourceMonitorDiagnosticSettingRead(d *pluginsdk.ResourceData, meta interf
 		return err
 	}
 
+	id.ResourceUri = normalizeMonitorDiagnosticSettingTargetResourceId(id.ResourceUri)
```

Comment:
i'm not sure why this is a function as its only used in a single place? can we inline it?

Reply:
done.